e2e/tests/desktop/login.spec.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const {test, expect} = require('@playwright/test')
+const config = require('../../config')
+const {generateUserCredentials} = require('../../scripts/utils.js')
+const {answerConsentTrackingForm} = require('../../scripts/pageHelpers.js')
+
+const GUEST_USER_CREDENTIALS = generateUserCredentials()
+/**
+ * Test that a user can login with passwordless login on mobile. There is no programmatic way to check the email,
+ * so we will check that the necessary API call is being made and expected UI is shown
+ */
+test('Verify passwordless login request', async ({page}) => {
+    let interceptedRequest = null
+    
+    await page.route('**/mobify/slas/private/shopper/auth/v1/organizations/*/oauth2/passwordless/login', (route) => {
+        interceptedRequest = route.request()
+        route.continue()
+    })
+
+    await page.goto(config.EXTRA_FEATURES_E2E_RETAIL_APP_HOME + '/login')
+    await answerConsentTrackingForm(page)
+
+    await page.locator('#email').scrollIntoViewIfNeeded()
+    await page.fill('#email', config.PWA_E2E_USER_EMAIL)
+
+    await page.getByRole('button', {name: 'Continue Securely'}).click()
+    
+    await page.waitForResponse('**/mobify/slas/private/shopper/auth/v1/organizations/*/oauth2/passwordless/login')
+    
+    expect(interceptedRequest).toBeTruthy()
+    expect(interceptedRequest.method()).toBe('POST')
+    
+    const postData = interceptedRequest.postData()
+    expect(postData).toBeTruthy()
+    
+    const params = new URLSearchParams(postData)
+    
+    expect(params.get('user_id')).toBe(config.PWA_E2E_USER_EMAIL)
+    expect(params.get('mode')).toBe('callback')
+    expect(params.get('channel_id')).toBe('RefArchGlobal')
```

Comment:
Move the siteID to config as well. Incase we need to move the site we're pointing to.